plugins/outputs/warp10/warp10.go

```diff
@@ -198,8 +198,42 @@ func boolToString(inputBool bool) string {
 	return strconv.FormatBool(inputBool)
 }
 
+/*
+Warp10 supports Infinity/-Infinity/NaN
+<'
+// class{label=value} 42.0
+0// class-1{label=value}{attribute=value} 42
+=1// Infinity
+'>
+PARSE
+
+<'
+// class{label=value} 42.0
+0// class-1{label=value}{attribute=value} 42
+=1// -Infinity
+'>
+PARSE
+
+<'
+// class{label=value} 42.0
+0// class-1{label=value}{attribute=value} 42
+=1// NaN
+'>
+PARSE
+*/
 func floatToString(inputNum float64) string {
-	return strconv.FormatFloat(inputNum, 'f', 6, 64)
+	var retv string
+	switch inputNum {
+	case math.NaN():
+		retv = "NaN"
+	case math.Inf(-1):
+		retv = "-Infinity"
+	case math.Inf(1):
+		retv = "Infinity"
+	default:
+		retv = strconv.FormatFloat(inputNum, 'f', 6, 64)
+	}
+	return retv
```

Comment:
Please check the linter issue (it's valid). It says: "no number is equal to `NaN` not even `NaN` itself. So please use the correct functions here (https://pkg.go.dev/math#IsNaN):
```suggestion
	switch {
	case math.IsNaN(inputNum):
		return "NaN"
	case math.IsInf(inputNum, -1):
		return "-Infinity"
	case math.IsInf(inputNum, 1):
		return "Infinity"
	}
	return strconv.FormatFloat(inputNum, 'f', 6, 64)
```